google-finance/src/adapter.ts

```diff
@@ -0,0 +1,37 @@
+import { Requester, Validator } from '@chainlink/external-adapter'
+import { Config, ExecuteWithConfig, ExecuteFactory } from '@chainlink/types'
+import { makeConfig } from './config'
+import { google } from 'boxhock_google-finance-data'
```

Comment:
I think this should a default import  `import google from 'boxhock_google-finance-data'`